bpf/lib/nodeport.h

```diff
@@ -924,8 +925,29 @@ nodeport_rev_dnat_ingress_ipv6(struct __ctx_buff *ctx, struct trace_ctx *trace,
 encap_redirect:
 	src_port = tunnel_gen_src_port_v6(&tuple);
 
+#ifdef IS_BPF_XDP
+	src_sec_identity = WORLD_ID;
+#endif
+
+#ifdef IS_BPF_HOST
+	src_sec_identity = SECLABEL;
```

Comment:
Ah interesting, is this (or equivalent existing logic) where host ends up having a default fallback of `HOST_ID`? Not a change for this commit or perhaps even PR but maybe we'll want to revisit this at some point.

Reply:
yep, this is the only part that's left. I definitely don't want to tackle it here, but it's something to chip away at in future PRs. There's a good chance that we can just pass a "proper" identity here, that was resolved earlier in the code flow.